cfme/base/ui.py

```diff
@@ -1349,6 +1349,9 @@ class ZoneDetails(CFMENavigateStep):
 
     prerequisite = NavigateToAttribute('appliance.server.zone.region', 'Zones')
 
+    def am_i_here(self):
```

Comment:
In general, to disable checking of am_i_here, `cfme.exceptions.displayed_not_implemented` should be set on the View class for the destination.

In this case, the issue is that multiple destinations are using the same view class.

I think a better course of action here would be to not re-use `ZoneDetailsView`, and to create a unique view class for `ZoneAdvancedView`, that implements its own is_displayed method.

With this returning `False`, the default behavior of wait_for_view will break, and navigation won't work cleanly for these destinations.

Reply:
Ok. I've implemented unique views for the tabs under Settings > Zone:


and for the tabs under Diagnostics > Zone:




Details and Zone > Advanced, as well as for the tabs in Diagnostics: